src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/params.py

```diff
@@ -453,6 +455,19 @@ def _configure_security_policy_storage_params(cmd):
 
     # TODO: use server default
 
+#####
+#           sql db transparent-data-encryption
+#####
+
+with ParametersContext(command='sql db tde') as c:
+    c.argument('database_name', options_list=('--database', '-d'), required=True,
```

Comment:
nit: since this is multi line function call, please put each arg on a separate line so that it looks pretty